pkg/operator/ceph/object/config.go

```diff
@@ -37,6 +37,10 @@ caps mon = "allow rw"
 caps osd = "allow rwx"
 `
 
+	caBundleVolumeName             = "rook-ceph-rgw-ca-bundle"
+	caBundleDir                    = "/etc/pki/tls/certs"
```

Comment:
RGW will pick up the certs found in this path already? No other params needed?

Reply:
We are using same paths in our WA for this stuff and it's working fine. I'll double check that.